### PR TITLE
Releasing 2.1.2

### PR DESCRIPTION
**Pull request by evert**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,14 @@
 ChangeLog
 =========
 
+2.1.2 (2019-06-25)
+------------------
+
+* #39: Some PHPDoc improvements.
+* PHPStan support.
+* Adopted the symfony php-cs-fixer standard.
```
Comment (staabm) [inline CHANGELOG.md:9]: Missing: Official Php7.3 support added

Comment (staabm) [reply CHANGELOG.md:9]: Also 7.2

Comment (evert) [reply CHANGELOG.md:9]: I think PHP 7.2 and 7.3 always worked?

Comment (evert) [reply CHANGELOG.md:9]: Added something!

**Pull request merged**